### PR TITLE
Add app version via mvn

### PR DESCRIPTION
**Pull request by ahalberkamp**
In this pr, the app version is set before compiling the resource, because the previous implementation on fetching the version on runtime didn`t work.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <java.version>1.8</java.version>
         <logStashLogbackEncoder.version>6.6</logStashLogbackEncoder.version>
-        <logback-json-classic.version>1.2.4-groovyless</logback-json-classic.version>
+        <logback-json-classic.version>1.2.4</logback-json-classic.version>
```
Comment (ahalberkamp) [inline pom.xml:20]:
```suggestion
            <logback-json-classic.version>1.2.4-groovyless</logback-json-classic.version>
```

File: pom.xml [hunk 2]
```diff
@@ -17,7 +17,7 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <java.version>1.8</java.version>
         <logStashLogbackEncoder.version>6.6</logStashLogbackEncoder.version>
-        <logback-json-classic.version>1.2.4-groovyless</logback-json-classic.version>
+            <logback-json-classic.version>1.2.4-groovyless</logback-json-classic.version>
```
Comment (ahalberkamp) [inline pom.xml:20]:
```suggestion
     <logback-json-classic.version>1.2.4-groovyless</logback-json-classic.version>
```

File: src/main/java-templates/utils/ProcessorSolutionInfo.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
+package com.commercetools.paymenttoorderprocessor.utils;
```
Comment (praveenkumarct) [inline src/main/java-templates/utils/ProcessorSolutionInfo.java:1]: Package name doesn't match with the path?

File: src/main/java-templates/utils/ProcessorSolutionInfo.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+
+
+public final class ProcessorSolutionInfo extends SolutionInfo {
+    private static final String LIB_NAME = "commercetools-payment-to-order-processor";
+    private String version = "${version}";
+
+    /**
+     * Extends {@link SolutionInfo} class of the JVM SDK to append to the User-Agent header with
+     * information of the commercetools-sync-java library
```
Comment (praveenkumarct) [inline src/main/java-templates/utils/ProcessorSolutionInfo.java:13]:
```suggestion
     * information of the commercetools-payment-to-order-processor library
```

File: src/main/java-templates/utils/ProcessorSolutionInfo.java [hunk 5]
```diff
@@ -0,0 +1,19 @@
+package com.commercetools.paymenttoorderprocessor.utils;
+
+import io.sphere.sdk.client.SolutionInfo;
+
+
+
```
Comment (praveenkumarct) [inline src/main/java-templates/utils/ProcessorSolutionInfo.java:6]: Can you delete these extra lines?

**Review by praveenkumarct — CHANGES_REQUESTED** [review]

File: src/main/java-templates/utils/ProcessorSolutionInfo.java (hunk 3 above)
Comment (ahalberkamp) [reply src/main/java-templates/utils/ProcessorSolutionInfo.java]: 6bfe054

File: src/main/java-templates/utils/ProcessorSolutionInfo.java (hunk 5 above)
Comment (ahalberkamp) [reply src/main/java-templates/utils/ProcessorSolutionInfo.java]: c86bd0caa8443f541300002e879526a8d3465d98

File: src/main/java-templates/utils/ProcessorSolutionInfo.java (hunk 4 above)
Comment (ahalberkamp) [reply src/main/java-templates/utils/ProcessorSolutionInfo.java]: 77996b2

**Review by praveenkumarct — APPROVED** [review]

**Pull request merged**